contributing.md

```diff
@@ -0,0 +1,106 @@
+# Enarx RFCs
+
+## Contributing
+
+### Do you need an RFC?
+
+Use an RFC to advocate substantial changes to the Aries ecosystem, where
+those changes need to be understood by developers who *use* Aries. Minor
+changes are not RFC-worthy, and changes that are internal in nature,
+invisible to those consuming Aries, should be documented elsewhere.
+
+### Preparation
+
+Before writing an RFC, consider exploring the idea on the
+[chat](https://gitter.im/enarx/community), on a daily meeting call
+(see the [wiki page on contributing](
+https://github.com/enarx/enarx/wiki/How-to-contribute),
+Encouraging feedback from maintainers is a good sign that you're on
+the right track.
+
+### How to propose an RFC
+
+  - Fork [the RFC repo](https://github.com/enarx/rfc).
+  - Pick a descriptive folder name for your RFC. Don't pick a number yet.
+  - Decide which parent folder is appropriate for your RFC.
+    If it is about a specific protocol or decorator or feature, its parent
+    should be /features; if it is about a concept that will be used in many
```

Comment:
I think adding backticks around '/features' so it appears as `/features` helps it stand out as a technical component (file path, code snippet, etc) of the repo.

Reply:
Very good suggestion, thanks.